spec/features/apply_for_help_with_fees_spec.rb

```diff
@@ -1,86 +1,108 @@
 # coding: utf-8
 require 'rails_helper'
 
+def find_button_by_label(label_text)
```

Comment:
Does Capybara's find_button not do the trick? It should take a Label text or a selector for its "locator", class and value can be passed as options
http://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Finders#find_button-instance_method